string-manipulation-1/src/get-words.js

```diff
@@ -0,0 +1,9 @@
+/* exported getWords */
+// use spaces to separate words within array
+function getWords(string) {
+  var word = string.split(' ');
+  if (string === '') {
+    word = [];
+  }
```

Comment:
This check could have been executed before `split` method so that it exits the function earlier for one less method to run :)